cmake/cpack_settings.cmake

```diff
@@ -1,14 +1,14 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
 
-set(CPACK_PACKAGE_NAME ${CCF_PROJECT})
+set(CPACK_PACKAGE_NAME "${CCF_PROJECT}_${COMPILE_TARGET}")
```

Comment:
I think the cmake of the sample we use in test_install_build will need adjusting. I suggest poking the daily canary to make sure.

Reply:
You can also test this locally. If you `sudo ninja install`, then you can run `test_install_build.sh` exactly as the CI job does (without including all the other tests). Worth confirming that with a few build configurations on both the CCF build and the app build.

Reply:
Tested `test_install_build.sh` with all three `COMPILE_TARGET` builds and the app build appears to work fine, also checked the package names when manually generated and they are as expected, poked the daily canary anyway to get more confidence

Reply:
This is surprising, because the sample code seems to special case unsafe only:

```
set(CCF_PROJECT "ccf")
if(USE_UNSAFE_VERSION)
  set(CCF_PROJECT "${CCF_PROJECT}_unsafe")
endif()
```